elements/pf-alert/pf-alert.ts

```diff
@@ -0,0 +1,317 @@
+import { LitElement, type TemplateResult, html, isServer, render } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { repeat } from 'lit/directives/repeat.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { getRandomId } from '@patternfly/pfe-core/functions/random.js';
+import { SlotController } from '@patternfly/pfe-core/controllers/slot-controller.js';
+import styles from './pf-alert.css';
+import '@patternfly/elements/pf-icon/pf-icon.js';
+import '@patternfly/elements/pf-button/pf-button.js';
+
+
+interface AlertAction {
+  action: 'dismiss' | 'confirm' | string;
+  text: string;
+}
+interface ToastOptions {
+  id?: string;
+  message: string | TemplateResult;
+  heading?: string;
+  state?: PfAlert['state'];
+  persistent?: boolean;
+  actions?: AlertAction[];
+}
+const ICONS = new Map(Object.entries({
+  neutral: 'minus-circle',
+  info: 'info-circle',
+  success: 'check-circle',
+  custom: 'bell',
+  cogear: 'cog',
+  warning: 'exclamation-triangle',
+  danger: 'exclamation-circle',
+  close: 'times',
+
+}));
+
+
+export class AlertCloseEvent extends Event {
+  constructor(public action: 'close' | 'confirm' | 'dismiss' | string) {
+    super('close', { bubbles: true, cancelable: true });
+  }
+}
+let toaster: HTMLElement;
+const toasts = new Set<Required<ToastOptions>>();
+
+
+@customElement('pf-alert')
+export class PfAlert extends LitElement {
+  static readonly styles: CSSStyleSheet[] = [styles];
+
+  public static async toast(options: Omit<ToastOptions, 'id'>): Promise<void> {
+    const {
+      message,
+      persistent = false,
+      heading = 'Success',
+      state = 'info',
+      actions = [],
+    } = options;
+    toaster ??= initToaster();
+    const id = getRandomId();
+    const toast = { actions, heading, message, state, id, persistent };
+    toasts.add(toast);
+    const { matches: motionOK } = window.matchMedia('(prefers-reduced-motion: no-preference)');
+    renderToasts();
+    const alert = toaster.querySelector(`#${id}`);
+    if (toaster.children.length && motionOK) {
+      flip(toaster);
+    }
+    await Promise.all(toaster.getAnimations().map(x => x.finished));
+    if (!persistent && alert) {
+      await Promise.all(alert?.getAnimations().map(x => x.finished) ?? []);
+      toasts.delete(toast);
+    }
+    renderToasts();
+  }
+
+
+  @property({ reflect: true })
+  state:
+    | 'warning'
+    | 'custom'
+    | 'neutral'
+    | 'info'
+    | 'success'
+    | 'danger'
+    | 'cogear' = 'neutral';
+
+  @property({ reflect: true }) variant?: 'alternate' | 'toast' | 'inline';
+
+  @property({ reflect: true, type: Boolean }) dismissable = false;
```

Comment:
dismissable is not a public api on the patternfly react alert

The general rule is this: unless there is a good reason to modify one of the react component's public APIs (e.g. to make it more appropriate for HTML/DOM usage),  `pf-alert` must adhere to the public API of the react component.